docs/index.md

```diff
@@ -1015,6 +1013,15 @@ Specify an explicit path to a [configuration file](#configuring-mocha-nodejs).
 
 By default, Mocha will search for a config file if `--config` is not specified; use `--no-config` to suppress this behavior.
 
+### `--node-option <name>, -n <name>`
+
+> _New in v9.1.0._
+
+For Node.js and V8 options. Mocha forwards these options to Node.js by spawning a new child-process.<br>
```

Comment:
To clear, can we specify name should be without `--` nor `--v8-`?

Reply:
I added the `--` info. I prefer to not mention `--v8-` as this is a Mocha specific quirk and may be confusing.